SurveyApp/Modules/Login/LoginViewController.swift

```diff
@@ -39,6 +39,11 @@ final class LoginViewController: UIViewController {
         super.viewDidLoad()
         output?.viewDidLoad()
     }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        beginAnimation()
```

Comment:
You shouldn't add the logic into view.
Like here, you decide when `viewDidAppear`, the view should begin the animation, let's move it to `Presenter` 👍